docs/types/schema.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Schema
+---
+
+# Schema
+
+Every GraphQL API has a schema and that is used to define all the
+functionalities for an API. A schema is defined by passing 3
+[object types](./object-types): `Query`, `Mutation` and `Subscription`.
+
+`Mutation` and `Subscription` are optional, meanwhile `Query` has to always be
+there.
+
+This is an example of a schema defined using Strawberry:
+
+```python
+import strawberry
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def hello(self) -> str:
+        return "Hello World"
+
+
+schema = strawberry.Schema(Query)
+```
+
+## API
+
+```python
+strawberry.Schema(
+    query: Type,
+    mutation: Type = None,
+    subscription: Type = None
+    validate_queries: bool = True
+)
+```
+
+<!-- TODO: add docs on directives, types, extensions and execution context class -->
+
+Creates a GraphQL schema
+
+- `query`: The root query type
+- `mutation`: The root mutation type
+- `subscription`: The root subscription type
+
+### `.execute()` (async)
+
+Executes a GraphQL operation against a schema (async)
+
+`execute(query, variable_values, context_value, root_value, operation_name, validate_queries)`
+
+| Parameter name   | Type                              | Description                                                                                            |
+| ---------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| query            | `str`                             | The document to be executed                                                                            |
+| variable_values  | `Optional[Dict[str, Any]] = None` | The variables for this operation                                                                       |
+| context_value    | `Optional[Any] = None`            | The value of the context that will be passed down to resolvers                                         |
+| root_value       | `Optional[Any] = None`            | The value for the root type that will passed down to root resolvers                                    |
+| operation_name   | `Optional[str] = None`            | The name of the operation you want to execute, useful when sending a document with multiple operations |
+| validate_queries | `Optional[bool] = True`           | This flag allows to disable query validation                                                           |
```

Comment:
Why is this `Optional` if the default isn't `None`?

Reply:
specifically the `validate_queries` field

Reply:
mistake :D